app/pathogen/arbovirus/dashboard/ArboStudyPopup.tsx

```diff
@@ -26,6 +26,25 @@ function pathogenTag(pathogen: string) {
     )
 }
 
+function pathogenFullString(pathogen: string) {
+    switch (pathogen) {
+        case "DENV":
+          return "Dengue Virus";
+        case "ZIKV":
+          return "Zika Virus";
+        case "CHIKV":
+          return "Chikungunya Virus";
+        case "YF":
+          return "Yellow Fever";
+        case "WNV":
+          return "West Nile Virus";
+        case "MAYV":
+          return "Mayaro Virus";
+        default:
+          return "Unknown";
+      }
+}
+
```

Comment:
Should this be defined elsewhere? Is this a good approach for what I am doing here?

Reply:
No problems with this approach imo. We could maybe move this to a helper seeing as we'll need it in other spots too but totally something we can hit later :+1: